starter-code/css/main.css

```diff
@@ -6,5 +6,6 @@ light blue: #1c5380
 
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
+  color: red;
   margin: 0;
 }
```

Comment:
CSS needs to be improved! If you don't understand it, we're here for questions ;D